README.md

```diff
@@ -1,35 +1,39 @@
+Timothy McGowen
+
 # User Interface Project II: Great Idea Web Page With Flexbox
 
-Now that you have experienced building a layout using `inline-block`.  Lets revisit the same HTML structure and refactor our first page with flex box module.  If you didn't finish yesterday's assignment, just start from scratch or use the solution code provided to you.
+Now that you have experienced building a layout using `inline-block`. Lets revisit the same HTML structure and refactor our first page with flex box module. If you didn't finish yesterday's assignment, just start from scratch or use the solution code provided to you.
 
-Once you have finished the home page using flex box module, go build the services page based on the [design file](design-files/services-desktop-design.png) provided to you.  This time you will need to structure your own HTML in addition to using flex box module.  You have been provided all the necessary images in the `img` folder.
+Once you have finished the home page using flex box module, go build the services page based on the [design file](design-files/services-desktop-design.png) provided to you. This time you will need to structure your own HTML in addition to using flex box module. You have been provided all the necessary images in the `img` folder.
 
 **Useful Resource: [CSS Tricks Flexbox Guide](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)**
 
 ## To Get Started
 
-* Fork the project to your GitHub account
-* Clone this project into a directory on your machine.
-* Open the code with your preferred integrated development environment (IDE).
-* Start on the objectives
+- Fork the project to your GitHub account
+- Clone this project into a directory on your machine.
+- Open the code with your preferred integrated development environment (IDE).
+- Start on the objectives
 
-[Click this link to learn how to use git on this project](https://youtu.be/8UQYTQzzNYM)  
+[Click this link to learn how to use git on this project](https://youtu.be/8UQYTQzzNYM)
 
 ## Objectives
 
-* [ ] Home Page Refactor:
-  * Note that the provided HTML is blank.  This is because I want you to use your code from yesterday if possible.  If you didn't finish, that's fine, just use the solution from yesterday
-  * Refactor the home page layout with flexbox where `inline-block` was used before.
-  * In your navigation elements, point the services `<a href=""></a>` to `services.html`
+- [ ] Home Page Refactor:
```

Comment:
when you finish items you can put an x inside the brackets and i see you have some really strict space rules haha